third_party/2and3/nmap/nmap.pyi

```diff
@@ -0,0 +1,127 @@
+import sys
+from typing import Callable, Dict, Iterator, List, Optional, Sequence, Text, Tuple, TypeVar
+
+if sys.version_info >= (3, 8):
+    from typing import TypedDict
+else:
+    from typing_extensions import TypedDict
+
+_T = TypeVar("_T")
+_Callback = Callable[[str, _Result], None]
+
+class _Result(TypedDict):
+    nmap: _ResultNmap
+    scan: Dict[str, PortScannerHostDict]
+
+class _ResultNmap(TypedDict):
+    command_line: str
+    scaninfo: _ResultNmapInfo
+    scanstats: _ResultNampStats
+
+class _ResultNmapInfo(TypedDict, total=False):
+    error: str
+    warning: str
+    protocol: _ResultNampInfoProtocol
+
+class _ResultNampInfoProtocol(TypedDict):
+    method: str
+    services: str
+
+class _ResultNampStats(TypedDict):
+    timestr: str
+    elapsed: str
+    uphosts: str
+    downhosts: str
+    totalhosts: str
+
+class _ResulHostUptime(TypedDict):
+    seconds: str
+    lastboot: str
+
+class _ResultHostNames(TypedDict):
+    type: str
+    name: str
+
+class _ResultHostPort(TypedDict):
+    conf: str
+    cpe: str
+    extrainfo: str
+    name: str
+    product: str
+    reason: str
+    state: str
+    version: str
+
+__last_modification__: str
+
+class PortScanner:
```

Comment:
Since this supports Python 2, we need to derive from `object`, like the implementation:

```suggestion
class PortScanner(object):
```

This also applies to the other classes below.

Reply:
Okay, taken.